db/migrate/20170403135631_set_visible_to_true_for_custom_attributes.rb

```diff
@@ -0,0 +1,13 @@
+class SetVisibleToTrueForCustomAttributes < ActiveRecord::Migration[5.0]
+  class CustomAttribute < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    CustomAttribute.update_all(:visible => true)
+  end
+
+  def down
```

Comment:
@enoodle For future reference, there is no need to include the `down` method if it is empty.